### ISSUE TITLE
controller hori real arcade pro

### ISSUE THREAD

**Issue by rudamaia** [ymartel06/GnGeo-Pi#5]
Guys... Im having a hard time trying to config the axes for this arcade stick.

> jstest js0
> Driver version is 2.1.0.
> Joystick (HORI CO.,LTD. REAL ARCADE Pro.V3) has 6 axes (X, Y, Z, Rz, Hat0X, Hat0Y)
> and 13 buttons (BtnX, BtnY, BtnZ, BtnTL, BtnTR, BtnTL2, BtnTR2, BtnSelect, BtnStart, BtnMode, BtnThumbL, BtnThumbR, ?).
> 
> Already did the buttons part flawlessly, just need to get the axis working.
> Axes:  0:-32767  1:-32767  2:-32767  3:-32767  4:     0  5:     0

My config:

> p1control A=J0B0,B=J0B3,C=J0B1,D=J0B2,START=J0B9,COIN=J0B8,UP=J0a5,DOWN=J0a5,LEFT=J0A4,RIGHT=J0A4,MENU=J0B12

**Comment by eXiteRIC** [comment]
Yep, me too. My one (HORI CO., LTD. REAL ARCADE Pro.EX with 11buttons. It seem can't run normally with axes. The ONLY button work is SELECT button.

Second, I want to ask. "gngeopi" display scanline is WONDERFUL, but i can't use my Generic-X pad.
So, i tried "pifba" and set > ( Display Effect: 0 none, 1 scanline + DisplayEffect=0 ) inside "gngeorc" file, The screen looking simular almost 90%. but screen CANT clearly as "gngeopi".  "gngeopi" & "pifba" this totally same or different when running under SCANLINES with NONE ? Any idea ?
